src/ai/message.ts

```diff
@@ -188,6 +188,23 @@ export class Messages extends FactoryBase {
     return messages;
   }
 
+  private async sendMessage(
```

Comment:
if it replaces createMessageAndRun(), createMessage(), createRun(), and waitForRun(), how about removing them?

Reply:
done! thanks!